android/app/src/main/AndroidManifest.xml

```diff
@@ -55,9 +55,27 @@
             </intent-filter>
         </activity>
 
-        <meta-data android:name="com.facebook.sdk.ApplicationId"
-            android:value="@string/facebook_app_id"/>
+        <!-- <meta-data android:name="com.facebook.sdk.ApplicationId"
+            android:value="@string/facebook_app_id"/> -->
+        <meta-data android:name="com.facebook.sdk.ApplicationId" android:value="@string/facebook_app_id"/>
+   	    <meta-data android:name="com.facebook.sdk.ClientToken" android:value="@string/facebook_client_token"/>
+        
         <activity android:name="com.facebook.FacebookActivity"
+            android:configChanges=
+                    "keyboard|keyboardHidden|screenLayout|screenSize|orientation"
+            android:label="@string/app_name" />
```

Comment:
@bhoomikshetty This line would create a exception. There is no longer a string with value name `app_name` inside `strings.xml` add rather :


```xml
<?xml version="1.0" encoding="utf-8"?>
<resources>
    <string name="app_name">Times Up</string>
    <string name="facebook_app_id">176011390810650</string>
    <string name="fb_login_protocol_scheme">fb176011390810650</string>
    <string name="facebook_client_token">7747884d502137dfa05db2bcb1234f77</string>
</resources>
```

Reply:
Fixed, Thankyou.